static/js/containers/ProfileFormContainer.js

```diff
@@ -0,0 +1,151 @@
+/* global SETTINGS: false */
+import React from 'react';
+
+import {
+  startProfileEdit,
+  updateProfile,
+  validateProfile,
+  clearProfileEdit,
+  fetchUserProfile,
+  saveProfile,
+} from '../actions';
+import {
+  setWorkHistoryEdit,
+  setWorkDialogVisibility,
+  setWorkDialogIndex,
+  setEducationDialogVisibility,
+  setEducationDialogIndex,
+  setEducationDegreeLevel,
+  setEducationDegreeInclusions,
+} from '../actions/ui';
+
+class ProfileFormContainer extends React.Component {
+  static propTypes = {
+    profile:    React.PropTypes.object,
```

Comment:
I set the `propTypes` here to DRY things up a bit (lots of shared stuff between `UserPage` and `ProfilePage`).